### PR TITLE
Add types for gaugeJS@1.3.3

### PR DESCRIPTION
**Pull request by fiskWasTaken**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

I am unable to match the name as the package uses an old-style package name with capital letters. Not sure what to do about that.

### CODE REVIEW

File: types/gaugejs/tslint.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```
Comment (plantain-00) [inline types/gaugejs/tslint.json]: For new packages, this file should only `"extends": "dtslint/dt.json"`

File: types/gaugejs/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# gaugeJS typings
+
+Typings for gauge.js, available as npm package `gaugeJS`.
+
+These definitions were created by hand based on the CoffeeScript source available at https://github.com/bernii/gauge.js.
+
+Only classes that are designed to be used are typed (the ones exported to window/exports).
```
Comment (plantain-00) [inline types/gaugejs/README.md]: If I'm right, this file isn't needed

**Review by plantain-00 — CHANGES_REQUESTED** [review]
Test file is missing.

**Comment by mhegazy** [comment]
Thanks for your contribution. This PR has failing CI tests and can not be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Pull request closed without merging**